### DISCUSSION TITLE
Disable toolTip from auto hovering the closest node in ScatterPlot Canvas

### DISCUSSION THREAD

**Discussion by ami-rashid** [plouc/nivo#1920] [Q&A]
Hi, 

I would like the tooltip to only appear when I am hovering over a node, not when I am hovering over blank space. It looks like the newer version of the scatterplot uses a voronoi mesh to determine which node to automatically hover over. Is there a way to disable this?

I have tried passing `useMesh={false}`, but that doesn't resolve the problem.

**Comment by SteveAtKlover** [comment]
@ami-rashid i know this is a super old post, but your solution works with the latest versions (at least on scatter plot). so thanks for that.
